documentation/asciidoc/accessories/iqaudio/introduction.adoc

```diff
@@ -1,30 +1,29 @@
-== Overview of Raspberry Pi Audio
+== Overview
  
-With our audio boards, you will be able to bring high quality audio to your existing Hi-Fi or Raspberry Pi based equipment and projects. We offer four different HAT (Hardware Attached on Top) options that will fit any Raspberry Pi using the 40-pin GPIO header.
+Raspberry Pi Audio Boards bring high quality audio to your existing hi-fi or Raspberry Pi-based equipment and projects. We offer four different Hardware Attached on Top (HAT) options that will fit any Raspberry Pi using the 40-pin GPIO header.
```

Comment:
Wikipedia says Hi-Fi is "Hi-Fi or HiFi" not "hi-fi"..? https://en.wikipedia.org/wiki/High_fidelity

Reply:
Needs adding to https://github.com/raspberrypi/style-guide/blob/master/style-guide.md :wink:

Reply:
Also does anyone really use the word "Hi-Fi" any more, I mean... we're not Boomers here?

Reply:
^hi-fi^audio maybe @LizUpton ?

Reply:
I'm just thrashing through some reference books and online references so I can make a sensible decision here - IT'S NOT IN THE STYLE GUIDE. Horrors.

Reply:
Oxford Style Guide sez:

![IMG-2928](https://user-images.githubusercontent.com/6738098/195837220-098b5431-7d19-4b8f-b03b-73f94f07bd65.jpg)

(And yes, I'm adding it to ours.)

Reply:
Bah, that's from the other place.

Reply:
Our style guide's the source of truth on this, and says hi-fi.